spec/integration/manifest.spec.yml

```
@@ -487,6 +500,25 @@ spec:
     - version
     - type
     - owner
+  allOf:
+    - if:
+        properties:
+          policy_templates:
+            maxItems: 1
+      then:
+        properties:
+          policy_templates_behavior:
+            enum:
+              - default
+            default: default
+      else:
+        properties:
+          policy_templates_behavior:
+            enum:
+              - integration_package_only
+              - policy_templates_only
+              - default
```

Options proposed above.
```suggestion
              - combined_policy
              - individual_policies
              - all
```